### DISCUSSION TITLE
View/export public from private key

### DISCUSSION THREAD

**Discussion by apenen** [isontheline/pro.webssh.net#402] [Ideas]
Hi!

Add an option to export public key from already save private key.

Thanks!

**Comment by isontheline** [comment]
Hi @apenen 👋

Good issue 🙏

Just one question : the option to export the public key from the private one is just intended to be exported on your device or to be sent to a SSH server?

**Comment by apenen** [comment]
Hi!

Both options would be helpful.

**Reply by isontheline** [reply]
Thanks for your feedback 🙏

I have update the tracking issue about key management : https://github.com/isontheline/pro.webssh.net/issues/46
